Server/src/server/dataCenter/models/card/Card.java

```diff
@@ -1,149 +1,149 @@
-        }
-        this.defaultAp = referenceCard.defaultAp;
-        this.defaultHp = referenceCard.defaultHp;
-        this.manaCost = referenceCard.manaCost;
-        this.price = referenceCard.price;
-        this.attackType = referenceCard.attackType;
-        this.range = referenceCard.range;
-    }
-
-    public Card(String name, String description, CardType cardType, ArrayList<Spell> spells, int defaultAp,
-			int defaultHp, int manaCost, int price, AttackType attackType, int range) {
-		this.name = name;
-		this.description = description;
-		this.type = cardType;
-		this.spells = spells;
-		this.defaultAp = defaultAp;
-		this.defaultHp = defaultHp;
-		this.manaCost = manaCost;
-		this.price = price;
-		this.attackType = attackType;
-		this.range = range;
-	}
-
-	public CompressedCard toCompressedCard() {
-        return new CompressedCard(
-                name, description, cardId,
-                spriteName, type, spells,
-                defaultAp, defaultHp, manaCost,
-                attackType, range, remainingNumber
-        );
-    }
-
-    @Override
-    public boolean equals(Object obj) {
-        if (!this.getClass().getName().equals(obj.getClass().getName())) return false;
-        Card card = (Card) obj;
-        return this.cardId.equalsIgnoreCase(card.cardId);
-    }
-
-    public String getName() {
-        return this.name;
-    }
-
-    public String getDescription() {
-        return this.description;
-    }
-
-    public String getCardId() {
-        return this.cardId;
-    }
-
-    public void setCardId(String cardId) {//TODO:Should be removed!
-        this.cardId = cardId;
-    }
-
-    public CardType getType() {
-        return this.type;
-    }
-
-    public List<Spell> getSpells() {
-        return Collections.unmodifiableList(spells);
-    }
-
-    public int getDefaultAp() {
-        return this.defaultAp;
-    }
-
-    public int getDefaultHp() {
-        return this.defaultHp;
-    }
-
-    public int getManaCost() {
-        return this.manaCost;
-    }
-
-    public AttackType getAttackType() {
-        return this.attackType;
-    }
-
-    public int getRange() {
-        return this.range;
-    }
-
-    public int getPrice() {
-        return price;
-    }
-
-    public int getRemainingNumber() {
-        return remainingNumber;
-    }
-
-    public void setRemainingNumber(int number) {
-        remainingNumber = number;
-    }
-
-    public void increaseRemainingNumber() {
-        remainingNumber++;
-    }
-
-    public void decreaseRemainingNumber() {
-        remainingNumber--;
-    }
-
-    public void addSpell(Spell spell) {
-        spells.add(spell);
-    }
-}
\ No newline at end of file
+//package server.dataCenter.models.card;
+//
+//import shared.models.card.CompressedCard;
+//import shared.models.card.spell.Spell;
+//import shared.models.card.AttackType;
+//import shared.models.card.CardType;
+//
+//import java.util.ArrayList;
+//import java.util.Collections;
+//import java.util.List;
+//
```

Comment:
Can this just be removed then if everything is commented out?

Reply:
Oops, thank you